common/beerocks/bwl/include/bwl/mon_wlan_hal.h

```diff
@@ -32,7 +33,13 @@ class mon_wlan_hal : public virtual base_wlan_hal {
         RRM_Beacon_Request_Status,
         RRM_Beacon_Response,
         RRM_STA_Statistics_Response,
-        RRM_Link_Measurement_Response
+        RRM_Link_Measurement_Response,
+        //CHANNEL_SCAN events
+        Channel_Scan_Triggered,
```

Comment:
`DCS_Scan_Triggered`?

Reply:
we separated DCS from channel scan.
only the task in controller remain with "DCS" naming.  statuses/events/params.
all rest is channel scan. database, CMDUs to upper layers (bml), lower layers(bwl)

This is also due to the fact there is already a definition for  channel  scan in easymesh R2.
we will use same terminology and structures.